spring-kafka-3/src/main/java/com/baeldung/spring/kafka/start/stop/consumer/UserEvent.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.spring.kafka.start.stop.consumer;
+
+public class UserEvent {
+    private String userEventId;
+    private long eventNanoTime;
+
+    public UserEvent(){}
```

Comment:
Do we need this?

Reply:
Have removed the other attributes from UserEvent, have just kept userEventId. 
Keeping this UserEvent pojo, to represent the message structure rather than just string.  Typically in real application we usually have standard object.